packages/fluentui/docs/src/prototypes/table/ResponsiveTable.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { TableCell, TableProps, Table } from '@fluentui/react-northstar';
+import calculateBreakpoints, { Column, Breakpoint } from '../../utils/calculateBreakpoints';
+import * as _ from 'lodash';
+
+interface ResponsiveTableProps extends TableProps {
+  config: Column[];
+  children: React.ReactElement<typeof Table>;
+  id?: string;
+}
+
+const ResponsiveTableContainer: React.FC<ResponsiveTableProps> = props => {
+  const { config, children: ChildrenTable } = props;
+
+  const [responsiveTableID] = React.useState(props.id || _.uniqueId(`${Table.className}__responsive-wrapper`));
+
+  const columns = ChildrenTable.props.header.items;
+
+  const [breakpoints, setBreakpoints] = React.useState<Breakpoint[]>();
+
+  // Generate the stylesheet once and for all
+  React.useLayoutEffect(() => {
+    setBreakpoints(calculateBreakpoints(config));
```

Comment:
Don't we need to have a prop for the `breakpoints` param here? If not, I would suggest removing that param from the `calculateBreakpoints`

Reply:
Done